src/Angular.js

```diff
@@ -893,7 +893,35 @@ var csp = function() {
   return (csp.isActive_ = active);
 };
 
+/**
+ * @ngdoc directive
+ * @name ng.directive:ngJq
+ *
+ * @element html
+ * @param {string} (optional) the name of the library available under `window` to be used for angular.element
```

Comment:
Weird, apparently, something got duplicated in the docs.  Some weird merge somewhere?  Anywho, that's been fixed and I've pushed again.  I'm still seeing the `grunt package` failing on doc generation though.